From 3dc2b4dd2e989440bc6055a5904e77b53581fa23 Mon Sep 17 00:00:00 2001
From: Valentin David <me@valentindavid.com>
Date: Sun, 26 Oct 2025 11:03:32 +0100
Subject: [PATCH 04/11] openssh: Use /etc/ssh as sysconfdir

All other distros use that. And there are expectations from systemd
for it to be configured so.

Also fix a /var/run
---
 elements/components/openssh.bst | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/elements/components/openssh.bst b/elements/components/openssh.bst
index dce91b9af..04758d472 100644
--- a/elements/components/openssh.bst
+++ b/elements/components/openssh.bst
@@ -16,6 +16,7 @@ runtime-depends:
 - components/sed.bst
 
 variables:
+  sysconfdir: '/etc/ssh'
   arch_options: ''
   (?):
   # ppc64le doesn't support --fzero-call-used-regs=used flag
@@ -29,6 +30,7 @@ variables:
     --with-pam
     --with-pam-service=sshd
     --without-zlib-version-check
+    --with-pid-dir=/run
     --with-security-key-builtin
     %{arch_options}
 
-- 
2.52.0

